### ISSUE TITLE
release not support h265?

### ISSUE THREAD

**Issue by pedoc** [henrypp/chromium#61]
test by https://www.h265files.com/embed-h265-video.php

**Comment by tianwen2976** [comment]
Yes, I not success too, it appears: "shm_open() failed: Function not implemented". Can someone guide me?
